README.md

```diff
@@ -47,11 +48,21 @@ Then you can visit:
 config-ui by url `http://YOUR-NODE-IP:30090`
 grafana by url `http://YOUR-NODE-IP:30091`
 
-## Update
+## Upgrade
+
+Note:
+If you're upgrading from DevLake v0.17.x or earlier versions to v0.18.x or later versions:
+
+1. Please copy the ENCODE_KEY value from /app/config/.env of the lake pod (eg: devlake-lake-0), and replace <ENCRYPTION_SECRET> of below upgrade cmd.
+
+2. You will encounter below error when upgrading, that's because the build-in grafana will be replaced by the official grafana dependency. 
```

Comment:
Can you update to:

2. You may encounter the below error when upgrading because the built-in grafana has been replaced by the official grafana dependency.